src/controllers/api.controller.ts

```diff
@@ -1,13 +1,16 @@
-import { Request, Response, NextFunction } from 'express';
-import { prisma } from '../db/prisma';
-import { Prisma } from 'prisma/prisma-client';
+import { Request, Response, NextFunction } from "express";
+import { prisma } from "../db/prisma";
 import {
   passToExpressErrorHandler,
   turnFalsyPropsToUndefined,
   poormansNormalizer,
-  makeClaimSubjectURL
-} from '../utils';
-import createError from 'http-errors';
+} from "../utils";
+import createError from "http-errors";
+
+import { ClaimDao, NodeDao } from "../dao/api.dao";
```

Comment:
what are ClaimDao and NodeDao conceptually, why did we pick the name "Dao" ?  it means to me a DAO organization, what does it mean in this context?